database.py

```diff
@@ -57,6 +57,18 @@ def getBot(self, name):
             return row[0], row[1], row[2], row[3]
         return -1, -1, -1
 
+    @property
+    def getBots(self):
+        """
+        Method used to get bots data from database ordered by ELO
+        @return: list of all rows from BOTS table
+        """
+        cursor = self.conn.execute("SELECT * from BOTS ORDER BY ELO DESC")
+        rows = []
+        for row in cursor:
+            rows.append(row)
+        return rows
```

Comment:
Why do you rewrite cursor to array ?
Can't you just return cursor as result like show here:
https://www.sqlitetutorial.net/sqlite-python/sqlite-python-select/

```python
cursor = self.conn.execute("SELECT * from BOTS ORDER BY ELO DESC")
return cursor.fetchall()
```

Reply:
You're right. I'm changing it.